src/Wt/WTable.h

```diff
@@ -197,6 +197,7 @@ class WT_API WTable : public WInteractWidget
   void expand(int row, int column, int rowSpan, int columnSpan);
   WTableCell *itemAt(int row, int column);
   void setItemAt(int row, int column, std::unique_ptr<WTableCell> cell);
+  void updateRowNumbers(int begin, int end);
```

Comment:
`firstRow` and `lastRow` is probably more clear than `begin` and `end`. It's also best that you throw in some documentation that clarifies that the `lastRow` is inclusive.